src/bootstrap.js

```diff
@@ -60,8 +60,15 @@ this.startup = function({webExtension}) {
   Services.prefs.clearUserPref(UA_ENABLE_PREF_NAME);
   Services.prefs.addObserver(UA_ENABLE_PREF_NAME, UAEnablePrefObserver, false);
 
-  overrider = new UAOverrider(UAOverrides);
-  overrider.init();
+  let messageManager = Components.classes["@mozilla.org/globalmessagemanager;1"]
```

Comment:
I usually avoid defining `Components.*` shorthands if they only get used once, but you are right, it visual makes a difference here. :)